src/overlays/actors/ovl_En_Owl/z_en_owl.h

```diff
@@ -4,12 +4,65 @@
 #include "ultra64.h"
 #include "global.h"
 
-struct EnOwl;
+typedef enum {
+    OWL_DEFAULT,
+    OWL_OUTSIDE_KOKIRI,
+    OWL_HYRULE_CASTLE,
+    OWL_KAKARIKO,
+    OWL_HYLIA_GERUDO,
+    OWL_LAKE_HYLIA,
+    OWL_ZORA_RIVER,
+    OWL_HYLIA_SHORTCUT,
+    OWL_DEATH_MOUNTAIN,
+    OWL_DEATH_MOUNTAIN2,
+    OWL_DESSERT_COLOSSUS,
+    OWL_LOST_WOODS_PRESARIA,
+    OWL_LOST_WOODS_POSTSARIA
+} EnOwlType;
 
-typedef struct EnOwl {
+typedef enum {
+    OWL_REPEAT,
+    OWL_OK,
+} EnOwlMessageChoice;
```

Comment:
are these enums used externally? type maybe for spawn params, but not the choice index.
id also put offset comments